lib/host.py

```diff
@@ -516,6 +516,9 @@ def disks(self):
         disks.sort()
         return disks
 
+    def raw_disk_is_available(self, disk):
+        return self.ssh_with_result(['blkid', '/dev/' + disk]).returncode == 2
+
```

Comment:
A docstring explaining what this new method wrt `disk_is_available` would be useful, as well as one for the latter for easier comparison.

It could also be good to add type hints on such new functions.

Reply:
Done